ACE/ace/OS_NS_Thread.cpp

```diff
@@ -3951,6 +4219,72 @@ ACE_OS::thr_join (ACE_thread_t waiter_id,
 }
 #endif /* ACE_HAS_VXTHREADS */
 
+#if defined (ACE_INTEGRITY) && !defined (ACE_HAS_PTHREADS)
+int
+ACE_OS::thr_join (ACE_hthread_t thr_handle,
+                  ACE_THR_FUNC_RETURN* status)
+{
+  ACE_UNUSED_ARG (status);
```

Comment:
You can comment out status as argument name, makes this unused arg not necessary